test/upgrade/upgradeeventing_test.go

```diff
@@ -0,0 +1,68 @@
+// +build upgradeeventing
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
+
+import (
+	"testing"
+
+	"go.uber.org/zap"
+	eventingupgrade "knative.dev/eventing/test/upgrade"
+	"knative.dev/operator/test/upgrade/installation"
+	pkgupgrade "knative.dev/pkg/test/upgrade"
+)
+
+func TestEventingUpgrades(t *testing.T) {
+	suite := pkgupgrade.Suite{
+		Tests: pkgupgrade.Tests{
+			PreUpgrade: []pkgupgrade.Operation{
+				EventingPreUpgradeTests(),
+				eventingupgrade.PreUpgradeTest(),
```

Comment:
Why do we need 2 implementations for each of those pre - post tests. Couldn't we just reuse the ones from eventing?

Reply:
Okay. I got it. Maybe it would be good to update the names to be clear that those are assertions for CR's for ex.: `EventingCRPreUpgradeTests()`